src/main/java/com/powsybl/openloadflow/sa/AbstractNetworkResult.java

```diff
@@ -73,9 +73,9 @@ private static ThreeWindingsTransformerResult createThreeWindingsTransformerResu
         LfBranch leg1 = network.getBranchById(LfLegBranch.getId(threeWindingsTransformerId, 1));
         LfBranch leg2 = network.getBranchById(LfLegBranch.getId(threeWindingsTransformerId, 2));
         LfBranch leg3 = network.getBranchById(LfLegBranch.getId(threeWindingsTransformerId, 3));
-        double i1Base = PerUnit.ib(leg1.getBus1().getNominalV());
-        double i2Base = PerUnit.ib(leg2.getBus1().getNominalV());
-        double i3Base = PerUnit.ib(leg3.getBus1().getNominalV());
+        double i1Base = leg1.isConnectedAtBothSides() ? PerUnit.ib(leg1.getBus1().getNominalV()) : Double.NaN;
```

Comment:
This is not the good fix. If buses are null, you must retrieve the nominal voltages through the transformer.